### ISSUE TITLE
weird error

### ISSUE THREAD

**Issue by vince62s** [sammous/spacy-lefff#24]
seems the word "DEV" triggers an error.
```#!/usr/bin/python3
import spacy
from spacy_lefff import LefffLemmatizer, POSTagger
nlp = spacy.load('fr_core_news_md')
pos = POSTagger()
french_lemmatizer = LefffLemmatizer(after_melt=True)
nlp.add_pipe(pos, name='pos', after='parser')
nlp.add_pipe(french_lemmatizer, name='lefff', after='pos')
doc = nlp(u"Paris est une ville très DEV ce jour.")
for d in doc:
    print(d.text, d.pos_, d._.melt_tagger, d._.lefff_lemma, d.tag_, d.lemma_)
```

`Traceback (most recent call last):
  File "test-lemm.py", line 11, in <module>
    doc = nlp(u"Paris est une ville très DEV ce jour.")
  File "/usr/local/lib/python3.6/dist-packages/spacy/language.py", line 402, in __call__
    doc = proc(doc, **component_cfg.get(name, {}))
  File "/usr/local/lib/python3.6/dist-packages/spacy_lefff/melt_tagger.py", line 242, in __call__
    beam_size=beam_size)
  File "/usr/local/lib/python3.6/dist-packages/spacy_lefff/melt_tagger.py", line 194, in tag_token_sequence
    best_sequence = sequences[-1][0]
IndexError: list index out of range
`

**Comment by sammous** [comment]
Thanks for the finding, I will have a look.

**Comment by Xelvoz** [comment]
I'm having the exact same error on my part as well. Any progress on it?

**Comment by sammous** [comment]
Should be fixed in version `0.3.7`.
